recipes/openmm-ml/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "openmm-ml" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmm/openmm-ml/archive/28b4fadd761221fbe085570faac3e95a2d0be4e0.tar.gz
+  sha256: 913c48641c5fdcbdc6d7ccf14cd33a20ad432c781455e331da86a7a8cc455330
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
```

Comment:
I think we also need PyTorch and torchani as dependencies.  Once https://github.com/openmm/openmm-ml/pull/21 is merged, we'll need NNPOps too.

Reply:
And also OpenMM-Torch.